### PR TITLE
Adds centre of geometry and centre of mass functionality

### PR DESCRIPTION
**Pull request by arm61**

### CODE REVIEW

File: docs/source/mda_COG.ipynb [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+     ]
+    }
+   ],
+   "source": [
+    "import numpy as np\n",
+    "import MDAnalysis as mda\n",
+    "import matplotlib.pyplot as plt\n",
+    "from kinisi.analyze import DiffusionAnalyzer\n",
+    "import warnings\n",
+    "warnings.filterwarnings(\"ignore\", category=UserWarning)"
```
Comment (arm61) [inline docs/source/mda_COG.ipynb:43]: Does an error get raised if you don't have this?

File: docs/source/mda_COG.ipynb [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+   "source": [
+    "plt.hist(diff.D.samples, density=True)\n",
+    "plt.axvline(diff.D.n, c='k')\n",
+    "plt.xlabel('$D$/cm$^2$s$^{-1}$')\n",
+    "plt.ylabel('$p(D$/cm$^2$s$^{-1})$')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
```
Comment (arm61) [inline docs/source/mda_COG.ipynb:284]: Can you delete the final empty cell please.

File: kinisi/parser.py [hunk 3]
```diff
@@ -52,7 +52,7 @@ class Parser:
     def __init__(self,
                  disp: np.ndarray,
                  indices: np.ndarray,
-                 framework_indices: np.ndarray,
+                 drift_indices: np.ndarray,
```
Comment (arm61) [inline kinisi/parser.py:55]: Can you remind me of the logic for renaming this again?

Comment (arm61) [reply kinisi/parser.py:55]: Nevermind, I have scrolled down and remember.

File: kinisi/parser.py [hunk 4]
```diff
@@ -688,52 +824,97 @@ def get_indices(structure: "MDAnalysis.universe.Universe", specie: str) -> Tuple
...
+        theta_bar = np.arctan2(zeta_bar, xi_bar)
+        new_s_coords = theta_bar / (2 * np.pi)
 
-        return structure, coords, (flat_indices, framework_indices)
+        new_coords = np.concatenate((new_s_coords, sq_coords[:, drift_indices]), axis=1)
+        new_indices = list(range(n_molecules))
+        new_drift_indices = list(range(n_molecules, n_molecules + len(drift_indices)))
+
+        if new_coords.shape[2] != 1:
+            new_coords = np.expand_dims(new_coords, axis=2)
```
Comment (arm61) [inline kinisi/parser.py:890]: Is there code duplication that we can avoid here?

**Review by arm61 — COMMENTED** [review]
Can you add the new tutorial so that it will appear on the documentation pages. This would involve editing the `notebooks.md` file and adding this tutorial to the list.

File: docs/source/mda_COG.ipynb (hunk 1 above)
Comment (jd15489) [reply docs/source/mda_COG.ipynb:43]: This suppresses warnings about missing atom mass data from MDAnalysis, although these warnings are very small: 
"UserWarning: Failed to guess the mass for the following atom types: SI warnings.warn("Failed to guess the mass for the following atom types: {}".format(atom_type))"

File: docs/source/mda_COG.ipynb (hunk 2 above)
Comment (jd15489) [reply docs/source/mda_COG.ipynb:284]: Done.

File: kinisi/parser.py (hunk 3 above)
Comment (jd15489) [reply kinisi/parser.py:55]: This is renamed so that framework_indices can be used as a keyword argument elsewhere.

**Comment by arm61** [comment]
I think this is ready to merge. Are you happy for me to go ahead @jd15489?

**Comment by jd15489** [comment]
I am happy for it to be merged.

**Pull request merged**